packages/lit-dev-content/src/components/litdev-example-controls.ts

```diff
@@ -15,9 +15,6 @@ export class LitDevExampleControls extends LitElement {
   static override styles = css`
     :host {
       display: flex;
-      flex-direction: row;
-      align-items: center;
-      justify-content: center;
```

Comment:
I personally prefer this with the centering especially when there are file tabs.

![Screen Shot 2021-09-07 at 5 22 40 PM](https://user-images.githubusercontent.com/15080861/132426144-8b2eb5f2-a182-4ec5-93c2-f01743309310.png)

Reply:
Oh yeah, definitely looks wrong here :D